src/Header/SearchBox/SearchBox.css

```diff
@@ -0,0 +1,30 @@
+.searchBox {
+    margin-top: 9px;
```

Comment:
nit. Keep similar rules next to each other. So `margin-right` right after `margin-top`